validation/standard/standard-versions-sepolia.toml

```diff
@@ -0,0 +1,14 @@
+["op-contracts/v1.4.0"] # Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
```

Comment:
I think due to the "proxy" issue, we may end up only needing one file (not split by mainnet and sepolia).

Reply:
These bytecodes are not currently correct, we need to get them from OP Mainnet.